packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -0,0 +1,410 @@
+          <Item key="one">Item One</Item>
+          <Item key="two" textValue="Item Two">
+            <Copy size="S" />
+            <Text>Item Two</Text>
+          </Item>
+          <Item key="three">Item Three</Item>
+        </ComboBox>
+      </div>
+    )
+  )
+  .add(
+    'menuTrigger: focus',
+    () => render({menuTrigger: 'focus'})
+  )
+  .add(
+    'menuTrigger: manual',
+    () => render({menuTrigger: 'manual'})
+  )
+  .add(
+    'isOpen',
+    () => (
+      <div>
+        <div>Note: Combobox needs focus to show dropdown.</div>
+        <ComboBox label="Combobox" isOpen onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
+          <Item key="one">Item One</Item>
+          <Item key="two" textValue="Item Two">
+            <Copy size="S" />
+            <Text>Item Two</Text>
+          </Item>
+          <Item key="three">Item Three</Item>
+        </ComboBox>
+      </div>
+    )
+  )
+  .add(
+    'defaultOpen',
+    () => (
+      <div>
+        <div>Note: Combobox needs focus to show dropdown.</div>
+        <ComboBox label="Combobox" defaultOpen onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
+          <Item key="one">Item One</Item>
+          <Item key="two" textValue="Item Two">
+            <Copy size="S" />
+            <Text>Item Two</Text>
+          </Item>
+          <Item key="three">Item Three</Item>
+        </ComboBox>
+      </div>
+    )
+  )
+  .add(
+    'inputValue (controlled)',
+    () => (
+      <ControlledValueComboBox inputValue="Test" />
+    )
+  )
+  .add(
+    'defaultInputValue (uncontrolled)',
+    () => render({defaultInputValue: 'Item'})
+  )
+  .add(
+    'selectedKey (controlled)',
+    () => (
+      <ControlledKeyComboBox selectedKey="4" />
+    )
+  )
+  .add(
+    'defaultSelectedKey (uncontrolled)',
+    () => render({defaultSelectedKey: 'two'})
+  )
+  .add(
+    'inputValue and selectedKey (controlled)',
+    () => (
+      <AllControlledComboBox selectedKey="2" inputValue="Kangaroo" />
+    )
+  )
+  .add(
+    'defaultInputValue and defaultSelectedKey (uncontrolled)',
+    () => render({defaultInputValue: 'Item Two', defaultSelectedKey: 'two'})
+  )
+  .add(
+    'inputValue and defaultSelectedKey (controlled by inputvalue)',
+    () => (
+      <ControlledValueComboBox inputValue="K" defaultSelectedKey="2" />
+    )
+  )
+  .add(
+    'defaultInputValue and selectedKey (controlled by selectedKey)',
+    () => (
+      <ControlledKeyComboBox defaultInputValue="Blah" selectedKey="2" />
+    )
+  )
```

Comment:
The reason they don't use the same render function is because they feature a set of buttons that alter props.selectedKey or props.inputValue respectively + come with text to show what those tracked values are. They also have different state trackers to simulate how a user would use the component with that specific controlled/uncontrolled prop combination